frontend/ui/widget/sortwidget.lua

```diff
@@ -396,7 +342,7 @@ function SortWidget:init()
 end
 
 function SortWidget:nextPage()
-    local new_page = math.min(self.show_page+1, self.pages)
+    local new_page = math.min(self.show_page + 1, self.pages)
```

Comment:
Is there a particular reason for these random changes?

Reply:
Shouldn't math signs be separated with spaces?

Reply:
We have a tacit agreement that it's part context, part personal preference. There's a difference between `page+1` (fairly clear) and `page*6*3*4*9` (hard to read).

I think we also agree that we rather wouldn't see some unrelated commit mentioned in git blame unless it were necessary.

Reply:
Reverted